### DISCUSSION TITLE
mouse wheel events

### DISCUSSION THREAD

**Discussion by get-0ut** [fancyapps/ui#418] [Q&A]
default wheel is zoom, I am trying to make this pan 
  ((Fancybox, wheel) => 'pan'),
  is thta the right code ?
  
  kindly share discord server invitation.

**Comment by fancyapps** [comment]
Hi,

It would be like this (https://jsfiddle.net/7zom5fyk/):

```js
Fancybox.bind('[data-fancybox="gallery"]', {
   wheel : "pan"
});  
```

The Discord link is in the top right corner of the home page.
